### ISSUE TITLE
run/fix any e2e tests related to settings menu reorg/nav changes as well as new project screens

### ISSUE THREAD

**Issue by jlmitch5** [ansible/awx#2329]
the uiToMergeDevel branch has two main enhancements: the new settings menu nav work, as well as an updated version of the projects list to our new style.

after the issues are fixed and that branch is merged into awx/devel, this card is about fixing any e2e tests that are impacted by that work.  this sort of issue will not be necessary in the future, as merging will be gated by the e2e tests running, but should the rest of the feature work land before all of the CI stuff is figured out, this card should be verified to determine "completion" of the feature

**Comment by dsesami** [comment]
this was broken out into two WIP PRs and that uiToMergeDevel will be deleted, the first PR is here: https://github.com/ansible/awx/pull/2356
